src/serverless-openapi-typescript.ts

```diff
@@ -1,11 +1,25 @@
-'use strict';
-const { upperFirst, mergeWith, set, isArray, get, isEmpty } = require('lodash');
-const tsj = require('ts-json-schema-generator');
-const yaml = require('js-yaml');
-const fs = require('fs');
-
-class ServerlessOpenapiTypeScript {
-  constructor(serverless, options) {
+import type Serverless from "serverless";
+import fs from "fs";
+import yaml from "js-yaml";
+import tsj,  {SchemaGenerator} from "ts-json-schema-generator";
+import {upperFirst, mergeWith, set, isArray, get, isEmpty } from "lodash" ;
+
+interface Options {
+  typescriptApiPath?: string;
+  tsconfigPath?: string;
+}
+
+export default class ServerlessOpenapiTypeScript {
+  private serverless: Serverless;
+  private readonly functionsMissingDocumentation: string[];
+  private readonly disable: boolean;
+  private hooks: { [hook: string]: () => {}};
+  private options: Options;
+  private typescriptApiModelPath: string;
+  private tsconfigPath: string;
+  private schemaGenerator: SchemaGenerator;
+
+  constructor(serverless: Serverless, options: Options) {
     this.serverless = serverless;
```

Comment:
```suggestion
  private readonly functionsMissingDocumentation: string[];
  private readonly disable: boolean;
  private hooks: { [hook: string]: () => {}};
  private typescriptApiModelPath: string;
  private tsconfigPath: string;
  private schemaGenerator: SchemaGenerator;

  constructor(private serverless: Serverless, private options: Options) {
```

Reply:
I thought we don't like it. what about 

remove this line? 
```
this.serverless = serverless;
```

TS compile this line automatically when we add the private/public to the contractor

Reply:
ee6a838 for now

Reply:
> I thought we don't like it  

Why not? It's a great TS feature.

Reply:
I agree. I love it